### ISSUE TITLE
Reachability with underapproximation

### ISSUE THREAD

**Issue by schillic** [JuliaReach/Reachability.jl#391]
See [Under-Approximating Reach Sets for Polynomial Continuous Systems](https://dl.acm.org/citation.cfm?id=3178133) and the generalization to systems with inputs in [Inner-Approximating Reachable Sets for Polynomial Systems with Time-Varying Uncertainties](https://arxiv.org/abs/1811.01086).

**Comment by mforets** [comment]
true. i've added some more text to the other issue (eg. "underapproximation") so it is easier to find. also i think one of the references you added is new and we should update #225 with that reference.
